terraform/workspace-variables/staging_app_env.yml

```diff
@@ -7,4 +7,4 @@ RAILS_MAX_THREADS: 5
 RAILS_SERVE_STATIC_FILES: true
 SERVICE_TYPE: web
 DATABASE_URL: postgres://postgres@localhost:5432
-REGISTER_AND_PARTNER_URL: "https://ecf-dev.london.cloudapps.digital/"
+REGISTER_AND_PARTNER_URL: "https://ecf-staging.london.cloudapps.digital/"
```

Comment:
Using this URL probably makes it easier to filter out these requests, most will go to s-manage-ear...